lib/MusicBrainz/Server/Edit/Medium/Edit.pm

```diff
@@ -57,6 +57,7 @@ sub release_id { shift->data->{release}{id} }
 
 has '+data' => (
     isa => Dict[
+        delete_tracklist => Optional[Bool],
```

Comment:
I don't think this needs to be stored in the edit data, actually (since we don't use it for anything once the edit is initialized). Though it doesn't look like the code stores it anyway 'cause of `delete $opts{delete_tracklist}`. So I'd just remove it from here and `change_fields`.